.github/workflows/deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - "master"
       - "hotfix/**"
+      - "feature/update-installer"
```

Comment:
Should be removed or we could add more generic branch name such as `installer/**` or `deploy/**`